test/ui/viewJobRoleTests.ts

```diff
@@ -0,0 +1,80 @@
+import { Builder, WebDriver } from 'selenium-webdriver';
+import webdriver from 'selenium-webdriver';
+import { expect } from 'chai';
+import * as chrome from 'selenium-webdriver/chrome';
+
+describe('Job Roles Test', function () {
+
+    let driver: WebDriver;
+
+    before(async function () {
+        driver = new Builder()
+            .forBrowser('chrome')
+            .setChromeOptions(new chrome.Options())
+            .build();
+
+    });
+
+    after(async function () {
+        try {
+            await driver.quit(); // Ensure this is awaited
+        } catch (error) {
+            console.error('Error quitting the driver:', error);
+        }
+    });
+
+    it('Page should load and title text should display', async function () {
+        const url: string = process.env.UI_TEST_URL || 'https://2qr8mnb3c3.eu-west-1.awsapprunner.com/openJobRoles';
+        await driver.get(url);
+        const title = await driver.findElement(webdriver.By.xpath('/html[1]/body[1]/a[1]/h1[1]')).getText();
+
+        expect(title).to.equal('Open Job Roles');
+    });
+
+    it('Job role table should populate', async function () {
+        const url: string = process.env.UI_TEST_URL || 'https://2qr8mnb3c3.eu-west-1.awsapprunner.com/openJobRoles';
+        await driver.get(url);
+        const data = await driver.findElement(webdriver.By.xpath('/html[1]/body[1]/a[1]/table[1]/tbody[1]/tr[2]/td[1]')).getText();
```

Comment:
This is not a good selector because it relies on the page not changing, let's use an id

Reply:
do this for all selectors